Kinvey/Kinvey/MIC.swift

```diff
@@ -130,9 +153,13 @@ open class MIC {
     }
     
     @discardableResult
-    class func login<U: User>(refreshToken: String, client: Client = sharedClient, completionHandler: User.UserHandler<U>? = nil) -> Request {
+    class func login<U: User>(
+        refreshToken: String,
+        client: Client = sharedClient,
+        completionHandler: User.UserHandler<U>? = nil
+    ) -> Request {
         let requests = MultiRequest()
-        let request = client.networkRequestFactory.buildOAuthGrantRefreshToken(refreshToken: refreshToken)
+        let request = client.networkRequestFactory.buildOAuthGrantRefreshToken(refreshToken: refreshToken, clientId: nil)
```

Comment:
@tejasranade question: how should we handle refresh tokens if you are using a `clientId`?